coriolis/wsman.py

```diff
@@ -49,7 +50,7 @@ def connect(self, url, username, auth=None, password=None,
             cert_key_pem=cert_key_pem)
 
     @classmethod
-    def from_connection_info(cls, connection_info):
+    def from_connection_info(cls, connection_info, timeout):
```

Comment:
This has the potential of breaking everywhere it's called. Please make it a kwarg which defaults to `DEFAULT_TIMEOUT`.